crypto/fipsmodule/ml_kem/ml_kem_ref/indcpa.c

```diff
@@ -190,6 +190,9 @@ void gen_matrix(ml_kem_params *params, polyvec *a, const uint8_t seed[KYBER_SYMB
       }
     }
   }
+  
+  // FIPS 203. Section 3.3 Destruction of intermidiate values.
+  OPENSSL_memset(buf, 0, sizeof(buf));
```

Comment:
Should these all be `OPENSSL_cleanse` to make sure the compiler doesn't optimize out the unused 0 value?

Reply:
yes, fixed!